txnkv/transaction/2pc.go

```diff
@@ -1140,6 +1141,69 @@ const (
 	stateClosed
 )
 
+// WriteAccessLevel represents the level of write access required to modify the value
```

Comment:
Is `phase` better than `level` in the definition?

Reply:
In my initial implementation I used `phase` too. However, I switched to `level` as it's more intuitive. `level` is self-explanatory, whereas `phase` might require additional context to understand.